### DISCUSSION TITLE
Errors with Radar-1080 and Sonarr-Anime running on Truenas Scale 23.10

### DISCUSSION THREAD

**Discussion by Noctalin** [Feramance/qBitrr#40] [Q&A] [answered]
What I want to achieve is to have qbitrr cleanup stalled or not downloading torrents which just stay for ever in qbittorrent sonarr and radarr.
Yet I get constant errors of Resource not found which seems to be happening when qbitrr does some kind of cleanup.
Is there something I misunderstand in the setup of qbitrr or the apps?
Something else I read is that Sonarr and Radarr need to be setup to tag downloads?
What is meant by that and how is this done correctly?

I understand that without using host path volume mapping I cannot get access to sonarr.db and radarr.db.
Is that a major problem or can I run qbitrr fine without it too?
Here is my configuration (changed it to txt because couldn't upload it): [config.txt](https://github.com/Feramance/qBitrr/files/13675924/config.txt)

I am running following as Apps (Kubernetes) on Truenas Scale 23.10.0.1 (Truecharts):
* Sonarr (3.0.10.1567)
* Radarr (5.1.3.8246)
* Prowlarr (1.11.1.4146)
* Overseer (1.33.2)
* qbittorrent (4.6.2)
* qbitrr (3.8.8)

A ZFS Pool with 7 Mirror vdevs of 2 HDD 7200 RPM.
An NFS-shared dataset **/mnt/mypool/home/media** which contains the following folders:
* torrents (this is where qbittorrent saves torrents by default, with subfolders radarr and tv-sonarr based on the categories)
* incomplete
* movies (for radarr)
* tv (for sonarr)

Every app has additional storage setup by using NFS share **/mnt/mypool/home/media** mounted always with same "Mount Path" **/mnt/media** containing the folders above.
Permissions are all set to user and group 568 as this is the default user for running apps from Truecharts.

qBittorrent is configured as follows:
![image](https://github.com/Feramance/qBitrr/assets/11031032/774747a7-dfa7-464f-b61a-fc0cf91b0a2f)

Here is a log file of what usually happens all the time:
[qbitrr_qbitrr-584799c457-7fzbj_qbitrr (1).log](https://github.com/Feramance/qBitrr/files/13675782/qbitrr_qbitrr-584799c457-7fzbj_qbitrr.1.log)

**Comment by Feramance** [comment, marked as answer]
So, a couple of things then. Firstly when setting up qbittorrent in Radarr/Sonarr you should always give them a category as shown below:
![image](https://github.com/Feramance/qBitrr/assets/38938175/c91eb1a9-a12d-4f9e-8102-b0fc70af276e)
This way Radarr/Sonarr will only ever read torrents that they are downloading, not have a possible mix up.

With regards to your second question, no it is not required to have access to the `db` files however, if you do want the `Search Missing` functionality to work then you need to map them with at least read permissions. I do plan on moving away from requiring access to the `db` files in the future, that will be update 4.x.x but for now this is the current functionality.

**Comment by Feramance** [comment]
Also for proper torrent handling I would recommend shying away from using a separate folder for incomplete torrents, found that qBitrr works better this way
